pydmd/bopdmd.py

```diff
@@ -59,6 +59,14 @@ class BOPDMDOperator(DMDOperator):
         previously-mentioned sorting methods is chosen depending on eigenvalue
         variance.
     :type eig_sort: {"real", "imag", "abs", "auto"}
+    :param eig_constraints: Set containing desired DMD operator eigenvalue
+        constraints. Currently available constraints are "stable", which
+        constrains eigenvalues to the left half of the complex plane,
+        "imag", which constrains eigenvalues to the imaginary axis, and
+        "conjugate_pairs", which enforces that eigenvalues are always
+        present with their complex conjugate. Note that constraints may be
+        combined if valid.
```

Comment:
Just to make this look nicer in the docs:
```suggestion
    :param eig_constraints: Set containing desired DMD operator eigenvalue
        constraints. Currently available constraints are `"stable"`, which
        constrains eigenvalues to the left half of the complex plane,
        `"imag"`, which constrains eigenvalues to the imaginary axis, and
        `"conjugate_pairs"`, which enforces that eigenvalues are always
        present with their complex conjugate. Note that constraints may be
        combined if valid.
```

Reply:
Done!